homeassistant/components/sunsynk/config_flow.py

```diff
@@ -0,0 +1,121 @@
+"""Config flow for Sunsynk integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from sunsynk.client import InvalidCredentialsException, Inverter, SunsynkClient
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN, STEP_USER
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_USERNAME): str,
+        vol.Required(CONF_PASSWORD): str,
+    }
+)
+
+
+class SunsynkHub:
```

Comment:
I guess this is also constructed from the scaffold script and doesn't seem necessary as you can also put it directly in the step